src/raygun.js

```diff
@@ -1023,25 +1023,9 @@ var raygunFactory = function(window, $, undefined) {
           _successCallback(xhr, url, data);
         }
       };
-    } else if (window.XDomainRequest) {
-      xhr.ontimeout = function() {
-        if (_enableOfflineSave) {
-          Raygun.Utilities.log('Raygun: saved locally');
-          offlineSave(url, data);
-        }
-      };
```

Comment:
please explain this regression?

Reply:
Fair.

This was a missed piece for another PR.
`window.XDomainRequest` has been obsoleted since IE8.

So I removed it from here as well.